docs/FAQ.md

```diff
@@ -104,7 +104,9 @@ files you want to keep around to match the pattern.
 
 If `$EDITOR` integration is important, something like `scratchpad.*` may be more
 helpful, as you can keep the filename extension intact (it
-matches `scratchpad.md`, `scratchpad.rs` and more).
+matches `scratchpad.md`, `scratchpad.rs` and more). Another option is to add a 
+directory to the global `.gitignore` which then stores all your temporary files
+and notes. To achieve this add `<your-directory>/` to `.git/info/exclude`.
```

Comment:
The global gitignore is `~/.git/ignore`, I think. 

(Optional) You could also link https://git-scm.com/docs/gitignore there. Finally, I would be slightly more concrete:

> For example, you could add `scratch/` to [`~/.git/ignore`](https://git-scm.com/docs/gitignore) and then store arbitrary files in `your-git-repo/scratch/`.